.github/workflows/ci.yml

```diff
@@ -64,8 +64,8 @@ jobs:
           auto_fix: true
           git_name: github-actions[bot]
           git_email: github-actions[bot]@users.noreply.github.com
-          commit_message: 'chore(lint): ${linter} fix'
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+          commit_message: '[ci] ${linter} fix'
+          github_token: ${{ secrets.NIGHTLY_PERSONAL_GITHUB_TOKEN }}
```

Comment:
I'm assuming this token isn't set up yet. Is there a reason why we can't use the other?

Reply:
Nope, we can revert!